app/controllers/work_days_controller.rb

```diff
@@ -0,0 +1,96 @@
+class WorkDaysController < ApplicationController
+  before_action :set_work_day, only: [:show, :edit, :update, :destroy]
+
+  def index
+    if params.has_key?(:month) && params.has_key?(:year)
+      @month = params[:month].to_i
+      @year = params[:year].to_i
+      @project = params.has_key?(:project) ? Project.find(params[:project].to_i) : nil
+      @time_sheet = time_sheet_for(@year, @month, @project)
+      @work_days = all_for(@year, @month, @project)
```

Comment:
Eine Variable mit einem @ ist eine Instanz-Variable, die auch an den View weitergereicht wird. Aus Gründen der Übersichtlichkeit und Einfachheit sollte man die Anzahl der Instanz-Variablen gering halten. In deinem Index-View brauchst du meines Erachtens nur @project, @time_sheet und @workdays. Somit kannst du @year und @month zu ganz normalen lokalen Variablen machen :)